### DISCUSSION TITLE
Typo/Bad wording in docs "FastAPI Path Parameters and Numeric Validations" page

### DISCUSSION THREAD

**Discussion by ccrenfroe** [fastapi/fastapi#6635] [Questions] [answered]
### First check

* [x] I added a very descriptive title to this issue.
* [x] I used the GitHub search to find a similar issue and didn't find it.
* [x] I searched the FastAPI documentation, with the integrated search.
* [x] I already searched in Google "How to X in FastAPI" and didn't find any information. -- *Unrelated*
* [x] I already read and followed all the tutorial in the docs and didn't find an answer. -- *Unrelated*
* [x] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic). -- *Unrelated*
* [x] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui). -- *Unrelated*
* [x] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc). -- *Unrelated*
* [x] After submitting this, I commit to one of:
    * Read open issues with questions until I find 2 issues where I can help someone and add a comment to help there.
    * I already hit the "watch" button in this repository to receive notifications and I commit to help at least 2 people that ask questions in the future.
    * Implement a Pull Request for a confirmed bug.

### Description

I believe there is a typo in the docs [here](https://fastapi.tiangolo.com/tutorial/path-params-numeric-validations/)! In the info box under the recap section, it states the following:

> Query, Path, and others you will see later are subclasses of a common Param class (that you don't need to use).
And all of them share the same all these same parameters of additional validation and metadata you have seen.

Seems a bit hard to read, no?

**Comment by Kludex** [comment]
What do you suggest?

What do you think about the following:

"`Query`, `Path`, and other~s~ **classes** you will see later are subclasses of `Param` ~class~ (that you don't need to use).
~And~ All of them share the same ~all these same~ parameters for ~of~ additional validation and metadata, **that** you have **already** seen."

Maybe removing also this "that you have already seen"? 

Either way, a PR **should** be welcome, the current text is really confusing. 😅

**Comment by ccrenfroe** [comment]
I think that would work great! That is much more clear to me. 😄 

So far, I think the text has been very nice, especially as someone new to developing APIs.

I'll gladly put up a PR for this if that is alright?

**Comment by Kludex** [comment]
> I'll gladly put up a PR for this if that is alright?

I don't have _merge_ rights, but I think it will be highly appreciated! :)

**Comment by ccrenfroe** [comment, marked as answer]
I put up a [PR](https://github.com/tiangolo/fastapi/pull/3219) for the doc change. I'll go ahead and close this issue since it has been addressed and a change proposed.

Thanks for your time and suggestion! :v:

**Comment by tiangolo** [comment]
Thanks for the help here @Kludex ! 👏  🙇 

Thanks for reporting back and closing the issue @ccrenfroe  👍

> Sorry for the long delay! 🙈  I wanted to personally address each issue/PR and they piled up through time, but now I'm checking each one in order.
